x-pack/plugins/observability/public/hooks/slo/use_clone_slo.ts

```diff
@@ -33,10 +34,10 @@ export function useCloneSlo() {
     {
       onMutate: async ({ slo, idToCopyFrom }) => {
         // Cancel any outgoing refetches (so they don't overwrite our optimistic update)
-        await queryClient.cancelQueries(['fetchSloList'], { exact: false });
```

Comment:
`{exact: false}` is the default behaviours. It means it will cancel all queries matching the queryKey including the nested ones. In this case, it would cancel all fetchSloList queries regardless of the filters, page, sorting applied.